src/lib/bootloader/autoyast_converter.rb

```diff
@@ -33,19 +33,28 @@ def import(data)
         bootloader = bootloader_from_data(data)
         return bootloader if bootloader.name == "none"
 
-        import_grub2(data, bootloader)
-        import_grub2efi(data, bootloader)
-        import_stage1(data, bootloader)
-        import_default(data, bootloader.grub_default)
-        import_device_map(data, bootloader)
-        import_password(data, bootloader)
-        # always nil pmbr as autoyast does not support it yet,
-        # so use nil to always use proposed value (bsc#1081967)
-        bootloader.pmbr_action = nil
-        cpu_mitigations = data.global.cpu_mitigations
-        bootloader.cpu_mitigations = CpuMitigations.from_string(cpu_mitigations) if cpu_mitigations
+        case bootloader.name
+        when "grub2", "grub2-efi"
+          import_grub2(data, bootloader)
+          import_grub2efi(data, bootloader)
+          import_stage1(data, bootloader)
+          import_default(data, bootloader.grub_default)
+          import_device_map(data, bootloader)
+          import_password(data, bootloader)
+          # always nil pmbr as autoyast does not support it yet,
+          # so use nil to always use proposed value (bsc#1081967)
+          bootloader.pmbr_action = nil
+          cpu_mitigations = data.global.cpu_mitigations
+          if cpu_mitigations
+            bootloader.cpu_mitigations = CpuMitigations.from_string(cpu_mitigations)
```

Comment:
should this be also shared with the systemd-boot option?

Reply:
Hm, what do you mean with this ?

Reply:
Those kernel parameters should be shared in the case of Grub2 and also in the case of sd-boot. Shouldn't?

Reply:
well, what I see so far is that sd-boot support in this PR as it is now does not allow kernel params modification and cpu_mitigations depends on it.

Reply:
for systemd-boot /etc/kernel/cmdline is used when creating new boot entries